modules/xmpp/JingleSessionPC.js

```diff
@@ -2031,7 +2031,14 @@ export default class JingleSessionPC extends JingleSession {
 
             // Add a new transceiver by adding a new mline in the remote description.
             remoteSdp.addMlineForNewLocalSource(MediaType.VIDEO);
-            this._renegotiate(remoteSdp.raw)
+
+            // Always initiate a responder renegotiate since the new m-line is added to remote SDP.
```

Comment:
why does it execute the responder flow? what if it's the p2p offerer?

Reply:
Even if the ep is the p2p offerer, the new m-line is always added to the remote SDP. Therefore the reneg has to start with a sRD call so that the a new m-line is added to the local SDP returned by createAnswer call.

Reply:
so it's allowed to use the two flows interchangeably? interesting...

Reply:
It doesn't matter whether you do sRD->cA->sLD or cO->sLD->sRD once the call has established. The initiator or responder flows matter only for the initial call establishment where one of the peers has to start by creating an offer.